src/playground.rs

```diff
@@ -0,0 +1,275 @@
+//! run rust code on the rust-lang playground
+
+use crate::{
+    api,
+    commands::{Args, Result},
+};
+
+use reqwest::header;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::str::FromStr;
+
+#[derive(Debug, Serialize)]
+struct PlaygroundCode {
+    channel: Channel,
+    edition: Edition,
+    code: String,
```

Comment:
Can you pass a `&'p str` here instead and make `PlaygroundCode` take a lifetime `PlaygroundCode<'p>`